README.md

```diff
@@ -9,7 +9,7 @@ The boards can be assembled by hand or purchased preassembled; the open-source
 IDE can be downloaded for free at https://www.arduino.cc/en/Main/Software
 
 * For more information, see the website at: https://www.arduino.cc/
-or the forums at: https://forum.arduino.cc/  
```

Comment:
Instead of adding the two trailing spaces back to this line, I think it would be better to do a space and then backslash so that we won't have the same problem if anyone trims trailing spaces on this file in the future.